lib/hipchat/room.rb

```diff
@@ -149,6 +149,7 @@ def send(from, message, options_or_notify = {})
           :message        => message,
           :message_format => options[:message_format] || 'html',
           :color          => options[:color],
+          :card           => options[:card],
```

Comment:
You need to update the other mocks, since now the card parameter is always being sent.